src/test/java/task2/DisplayOfTheTaskTest.java

```diff
@@ -0,0 +1,91 @@
+package task2;
+
+import lambdastreams.task2.Book;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+public class DisplayOfTheTaskTest {
+    Book book1 = new Book("myBook", 5);
+    Book book2 = new Book("myBook", 5);
+
+    List<Book> listOfRandomBooks = List.of(new Book("Zzq", 23), new Book("Privet", 26),
```

Comment:
Please rename this variable to just "listOfBooks" since they are not random, and always the same

Reply:
Done